lib/selectors/thread-selectors.js

```diff
@@ -173,6 +173,19 @@ const unreadCount: (state: BaseAppState<*>) => number = createSelector(
     )(threadInfos),
 );
 
+const unreadBackgroundCount: (
+  state: BaseAppState<*>,
+) => number = createSelector(
+  (state: BaseAppState<*>) => state.threadStore.threadInfos,
+  (threadInfos: { [id: string]: RawThreadInfo }): number =>
+    _flow(
+      _filter(threadInChatList),
+      _filter(thread => !thread.currentUser.subscription.home),
+      _filter('currentUser.unread'),
```

Comment:
Nit: I prefer not to use this kind of string shorthands. Flow would never catch it if `RawThreadInfo` type would change in the future

Reply:
I did this like it was in `unreadCount`, but I also prefer not to use them

Reply:
Actually, `thread.currentUser.subscription.home` was supposed to be done the same way, but for now it's reversed value so I did it traditionally

Reply:
Good call @zrebcu411. I have a lot of Lodash usage across this project that I would prefer to decrease. In particular these string shorthands can't be typechecked.

I also noticed that the first two conditions here are basically `threadInBackgroundChatList`. What if we updated this function to look like this:

```js
import { values } from '../utils/objects';

import { threadInBackgroundChatList } from '../shared/thread-utils';

const unreadBackgroundCount: (
  state: BaseAppState<*>,
) => number = createSelector(
  (state: BaseAppState<*>) => state.threadStore.threadInfos,
  (threadInfos: { [id: string]: RawThreadInfo }): number => values(threadInfos).filter(
    threadInfo => threadInBackgroundChatList(threadInfo) && threadInfo.currentUser.unread,
  ).length,
);
```